### PR TITLE
WINDUPRULE-723 Quarkus Release 1.12 - run.jar issue

### PR DESCRIPTION
**Pull request by PhilipCattanach**

### CODE REVIEW

**Comment by PhilipCattanach** [comment]
@m-brophy this is very similar to WINDUPRULE-715 using the same criteria. Just a different issue and link generated. Thanks.

File: rules-reviewed/quarkus1/quarkus1-11/quarkus1-12-run-jar.windup.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="quarkus1-12-run-jar"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset gives hints for upgrading to Quarkus 1.12 - Fast-jar as default
```
Comment (m-brophy) [inline rules-reviewed/quarkus1/quarkus1-11/quarkus1-12-run-jar.windup.xml:6]: description refers to fast-jar

File: rules-reviewed/quarkus1/quarkus1-11/quarkus1-12-run-jar.windup.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <sourceTechnology id="quarkus1" versionRange="(,11]"/>
+        <targetTechnology id="quarkus1" versionRange="[12,)"/>
+    </metadata>
+    <rules>
+        <rule id="quarkus1-12-run-jar-00000">
+            <when>
+                <file filename="application.properties"/>
+            </when>
+            <perform>
+                <hint title="Starting the application with the -run.jar" effort="1" category-id="potential">
```
Comment (m-brophy) [inline rules-reviewed/quarkus1/quarkus1-11/quarkus1-12-run-jar.windup.xml:23]: stray hyphen in the title

File: rules-reviewed/quarkus1/quarkus1-11/tests/quarkus1-12-run-jar.windup.test.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+            <rule id="quarkus1-12-run-jar-00000-test">
+                <when>
+                    <not>
+                        <iterable-filter size="1"> 
+                            <hint-exists message="The biggest change here is that to start*"/>
+                        </iterable-filter>  
+                    </not>
+                </when>
+                <perform>
+                    <fail message="[quarkus1-12-run-jar-00000] The fast-jar hint was not found!" />
```
Comment (m-brophy) [inline rules-reviewed/quarkus1/quarkus1-11/tests/quarkus1-12-run-jar.windup.test.xml:19]: fail message refers to fast-jar

**Review by m-brophy — CHANGES_REQUESTED** [review]

**Review by m-brophy — APPROVED** [review]

**Pull request merged**